src/js/pre-need/utils/helpers.js

```diff
@@ -34,6 +34,20 @@ export function claimantHeader({ formData }) {
 }
 
 export function transform(formConfig, form) {
+  // Rename hasCurrentlyBuried data
+  const hasCurrentlyBuriedData = (application) => {
+    if (isVeteran({ application })) {
+      /* eslint-disable no-param-reassign */
+      application.hasCurrentlyBuried = application.veteranHasBuried;
```

Comment:
I think you could put these two changes in the existing functions (populateVeteranData and populateSponsorData)

Reply:
Good idea @jbalboni, these have been relocated.